### PR TITLE
Added NationBuilder to vendors.yml

### PR DESCRIPTION
**Pull request by asayler**

### CODE REVIEW

File: _data/vendors.yml [hunk 1]
```diff
@@ -132,11 +131,19 @@
   pricing_source: https://mattermost.com/pricing/
   updated_at: 2018-12-10
 
+- name: NationBuilder
+  url: https://nationbuilder.com
+  base_pricing: $29 per mmonth
```
Comment (robchahin) [inline _data/vendors.yml]:
```suggestion
  base_pricing: $29 per month
```

File: _data/vendors.yml [hunk 2]
```diff
@@ -132,11 +131,19 @@
...
+  sso_pricing: Call Us! (over $199/month)
+  percent_increase: 7x+
```
Comment (robchahin) [inline _data/vendors.yml]:
```suggestion
  percent_increase: 586%++[^nationbuilder] 
```

File: _data/vendors.yml [hunk 3]
```diff
@@ -132,11 +131,19 @@
...
   updated_at: 2018-12-10
 
+- name: NationBuilder
+  url: https://nationbuilder.com
+  base_pricing: $29 per mmonth
+  sso_pricing: Call Us! (over $199/month)
+  percent_increase: 7x+ 
+  pricing_source: https://nationbuilder.com/pricing
+  updated_at: 2019-02-09
+
```
Comment (robchahin) [inline _data/vendors.yml]:
```suggestion
  footnotes: "[^netbuilder]: To upgrade one plan level is represented by this price increase, but to get single sign-on, you actually have to go up another plan, which is 'Call Us' pricing."

```

**Comment by asayler** [comment]
Thanks for the cleanup!

**Pull request merged**